TL improvements (#351)
07th-mod/higurashi-console-arcs: Update/omot_014.txt

			NULL, "\"I was going to call her.", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101310", 256, TRUE);
	OutputLine(NULL, "…その前に、お前から電話があったから」",

> 			NULL, " ...But before doing that, I got a call from you.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...But before I could, I got a call from you.\"", Line_Normal);
```

//rみんなにいい顔をしようとして、思ったことをがんがん言ってやればいいのにそれが出来ない、情けないやつ。そんな魅音を御輿に担いでる園崎家、雛見沢の連中。まったく、おめでたい。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "みんなにいい顔をしようとして、思ったことをがんがん言ってやればいいのにそれが出来ない、情けないやつ。そんな魅音を御輿に担いでる園崎家、雛見沢の連中。まったく、おめでたい。",

> 			NULL, "Mion is trying put a good face for everyone, so she can't say frankly what she thinks. How miserable. Such a girl gets paraded around by the Sonozaki family and the people of Hinamizawa. What a hopeless fool.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mion is trying put on a good face for everyone, so she can't speak frankly what she thinks. How miserable. She gets paraded around by the Sonozaki family and the people of Hinamizawa. What a hopeless fool.", Line_Normal);
```

//r圭ちゃんが割って入ってきても、熱いね～、なんて茶化しながら平然と流すことが出来たのに。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "圭ちゃんが割って入ってきても、熱いね～、なんて茶化しながら平然と流すことが出来たのに。",

> 			NULL, "Even if Kei-chan butts in on me and her, I'd be able to calm myself by teasing him like \"you're worked up~.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even if Kei-chan butts in on me and her, I'd be able to calm myself by teasing him like \"this is getting steamy~.\"", Line_Normal);
```

//r私は、涙を枕元のティッシュで拭って立ち上がる。そして、鼻声を誤魔化すようにわざと、不機嫌さを押し出して言った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "私は、涙を枕元のティッシュで拭って立ち上がる。そして、鼻声を誤魔化すようにわざと、不機嫌さを押し出して言った。",

> 			NULL, "I grabbed a tissue from by my bedside to wipe away my tears, then stood up. And then, I spoke in a deliberately distorted, guttural voice to vent my displeasure, replacing tearful voice.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I grabbed a tissue from by my bedside to wipe away my tears, then stood up. And then I spoke up, replacing my tearful voice with a deliberately distorted, guttural voice to vent my displeasure.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300714", 256, TRUE);
	OutputLine(NULL, "「…いい、婆っちゃになんて言われても…」",

> 			NULL, "\"...I don't mind, no matter how much Granny tell off me...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...I don't mind, no matter what Granny has to say...\"", Line_Normal);
```

	ClearMessage();

//詩音rvS14/06/210600498.「あのね、そんなこと言われても私が困ります。kvS14/06/210600499.どうせバァさんのことだから、また私がお姉を足止めしたんじゃないかって考えるでしょうし。kvS14/06/210600500.そしたら結局、私が悪者じゃないですか」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600498", 256, TRUE);
	OutputLine(NULL, "「あのね、そんなこと言われても私が困ります。",

> 			NULL, "\"Well, even if you say that, it's rather a bother for me.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Well, even if you say that, it's a bother for me.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300714", 256, TRUE);
	OutputLine(NULL, "「…いい、婆っちゃになんて言われても…」",

> 			NULL, "\"...I don't mind, how much Granny tells off me...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...I don't mind, no matter what Granny has to say...\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300719", 256, TRUE);
	OutputLine(NULL, "「……私、悟史のために私のできることをやった、",

> 			NULL, "\"...All this time, I always thought I did all I could for Satoshi,", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...All this time, I thought I did all I could for Satoshi,", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300721", 256, TRUE);
	OutputLine(NULL, "「婆っちゃとか園崎の人間にも色々行き先聞いて回ったし、大石に疑われたときは悟史が捕まらないようにって、",

> 			NULL, "\"I asked Granny and the Sonozaki Group where he had gone. And when Ooishi started suspecting Satoshi, so that he didn't get caught,", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I asked Granny and the Sonozaki Group where he went. And when Ooishi started suspecting Satoshi...", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300722", 256, TRUE);
	OutputLine(NULL, "…お父さんに頼んで動いてもらった」",

> 			NULL, " ...I begged Dad to hush up.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I begged Dad do something to keep him from getting caught.\"", Line_Normal);
```

	ClearMessage();

//魅音rvS14/03/210300723.「だから、悟史がいなくなって、どこに行ったのかわからないって知ったときは、kvS14/03/210300724.…正直もうどうしようもないって、自分で勝手に納得させてた…」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300723", 256, TRUE);
	OutputLine(NULL, "「だから、悟史がいなくなって、どこに行ったのかわからないって知ったときは、",

> 			NULL, "\"So, when I realized Satoshi disappeared and I couldn't find out his whereabouts...", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"So, when I realized Satoshi disappeared and I couldn't find out where he was...", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300724", 256, TRUE);
	OutputLine(NULL, "…正直もうどうしようもないって、自分で勝手に納得させてた…」",

> 			NULL, " ...honestly, I was convinced that there wasn't anything more I could do...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " honestly, I was convinced that there wasn't anything more I could do...\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300723", 256, TRUE);
	OutputLine(NULL, "「だから、悟史がいなくなって、どこに行ったのかわからないって知ったときは、",

> 			NULL, "\"So, when I realized Satoshi disappeared and there was no way to find out his whereabouts,", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"So, when I realized Satoshi disappeared and there was no way to find out where he was...", Line_WaitForInput);
```

> 			NULL, "\"So, when I understood Satoshi disappeared and that there was no way to find out his whereabouts,", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"So, when I realized Satoshi disappeared and there was no way to find out where he was...", Line_WaitForInput);
```

			NULL, "\"That was when I learned about your relationship with Satoshi, Shion.", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300726", 256, TRUE);
	OutputLine(NULL, "二人の仲を認めてあげてって、悟史はとてもいい子だよって、婆っちゃにも園崎のみんなにも一生懸命、説得した」",

> 			NULL, " I did the best I could to convince Granny and the Sonozakis \"please allow her to associate with Satoshi, he was a good kid.\"\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I did the best I could to convince Granny and the Sonozakis that Satoshi was a good kid, and to allow you to associate with him.\"", Line_Normal);
```

> 			NULL, " I did the best I could to convince Granny and the Sonozakis, saying, \"Please allow her to associate with Satoshi, he's a good kid.\"\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I did the best I could to convince Granny and the Sonozakis that Satoshi was a good kid, and to allow you to associate with him.\"", Line_Normal);
```

	ClearMessage();

//魅音rvS14/03/210300727.「…でもみんな、うん、ってなかなか言ってくれなくて…kvS14/03/210300728.悔しくて、悲しくて…詩音たちにごめんって、いつも思ってた…」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300727", 256, TRUE);
	OutputLine(NULL, "「…でもみんな、うん、ってなかなか言ってくれなくて…",

> 			NULL, "\"...But everyone, well, they had a hard time saying sure...", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...But everyone, well, they had a hard time saying okay...", Line_WaitForInput);
```

			NULL, "\"That was when I learned about your relationship with Satoshi, Shion.", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300726", 256, TRUE);
	OutputLine(NULL, "二人の仲を認めてあげてって、悟史はとてもいい子だよって、婆っちゃにも園崎のみんなにも一生懸命、説得した」",

> 			NULL, " I did the best I could to convince Granny and the Sonozakis, saying \"please allow her to associate with Satoshi, he's a good kid.\"\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I did the best I could to convince Granny and the Sonozakis that Satoshi was a good kid, and to allow you to associate with him.\"", Line_Normal);
```

			NULL, "\"It wouldn't just be with all my might. I'd be desperate, willing to throw my life away...! ", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300735", 256, TRUE);
	OutputLine(NULL, "　それでもだめなやつがいたら、最後は殺してでもっ…！」",

> 			NULL, " Still, if some guys didn't accept, I'd go so far as killing them in the end...!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Still, even if people didn't accept it, I'd go so far as killing them in the end...!\"", Line_Normal);
```

> 			NULL, " Still, even if anyone didn't accept, I'd go so far as killing them in the end...!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Still, even if people didn't accept it, I'd go so far as killing them in the end...!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600509", 256, TRUE);
	OutputLine(NULL, "「全て終わって、どうしようもなくなってから謝罪されても悟史くんは帰ってこないんだッッッ！！！」",

> 			NULL, "\"It's all over. No matter how much you apologize after there was nothing we could do, it won't bring Satoshi-kun back!!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"It's all over. No matter how much you apologize for doing nothing, it won't bring Satoshi-kun back!!!\"", Line_Normal);
```

> 			NULL, "\"No matter how much you apologize after it's all over and there is nothing we could do, it won't bring Satoshi-kun back!!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"It's all over. No matter how much you apologize for doing nothing, it won't bring Satoshi-kun back!!!\"", Line_Normal);
```

//r魅音に甘えて、自分で動こうともせず、ただ周りや色んなものを恨んで、憎んで、…駄々をこねていただけなんだから……！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "魅音に甘えて、自分で動こうともせず、ただ周りや色んなものを恨んで、憎んで、…駄々をこねていただけなんだから……！！",

> 			NULL, "I was dependent to Mion and unwilling to move, just holding a grudge and feeling hatred toward everything around me... and throwing a tantrum...!!", Line_Normal);

Reviewer: ```suggestion
			NULL, "I was dependent on Mion and unwilling to move, just holding a grudge and feeling hatred toward everything around me... and throwing a tantrum...!!", Line_Normal);
```

			NULL, "\"It's getting late... You can stay here tonight.", Line_WaitForInput);
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600535", 256, TRUE);
	OutputLine(NULL, "……一緒に、バァさんに怒られましょう…？」",

> 			NULL, " ...Let's be scolded by Granny, together...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...Let's make Granny angry, together...\"", Line_Normal);
```

//r話を聞いてほっとすると同時に、俺は心底満たされた思いだった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "話を聞いてほっとすると同時に、俺は心底満たされた思いだった。",
			NULL, "I felt relieved and deeply satisfied to hear that things worked out.", Line_Normal);
	ClearMessage();

//詩音rvS14/06/210600538.「んで、せっかくだからってことで…実はついさっきまで、一緒だったんですよ。kvS14/06/210600539.二人して学校サボって、どーでもいい話で盛り上がっていました。kvS14/06/210600540.あっはっはっは…！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600538", 256, TRUE);
	OutputLine(NULL, "「んで、せっかくだからってことで…実はついさっきまで、一緒だったんですよ。",

> 			NULL, "\"Well, we got an opportunity to be together, so... we were actually still together until just a moment ago.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Well, since we had the opportunity to be together... we were actually still together until just a moment ago.", Line_WaitForInput);
```

//rとはいえ、それほどに嬉しかったのだという詩音の気持ちは、十分すぎるほど良くわかる。ならば、それに応えてこそ男というものだ！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "とはいえ、それほどに嬉しかったのだという詩音の気持ちは、十分すぎるほど良くわかる。ならば、それに応えてこそ男というものだ！",

> 			NULL, "Nevertheless, I had more than enough sympathy with Shion's happy feelings as much as she showed me. In which case, the only proper response is to take it like a man!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Nevertheless, I had more than enough sympathy for the happy feelings Shion had shown me. In which case, the only proper response is to take it like a man!", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600547", 256, TRUE);
	OutputLine(NULL, "「もらったんじゃないですよ？",

> 			NULL, "\"That doesn't mean she gave as a present, okay?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I wouldn't say she gave it to me, okay?", Line_WaitForInput);
```

//rなんだかオウム返しばかりだな、と思いながらも、聞き返す。詩音はちょっと呆れたように肩をすくめて、首を何度か振って答えた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "なんだかオウム返しばかりだな、と思いながらも、聞き返す。詩音はちょっと呆れたように肩をすくめて、首を何度か振って答えた。",

> 			NULL, "I repeated the word back at her, thinking I'm like a parrot. Shion shrugged her shoulders like she was a bit surprised, then shook her head several times and answered.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I repeated the word back at her, feeling like a parrot. Shion shrugged her shoulders like she was a bit surprised, then shook her head several times and answered.", Line_Normal);
```

//r…なんでも園崎お魎は、５年前に起きたダム工事反対運動にまつわる、犬飼建設大臣孫誘拐事件の教唆を行ったとして見られているそうだが、実際には何もしていなかった、とのこと。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…なんでも園崎お魎は、５年前に起きたダム工事反対運動にまつわる、犬飼建設大臣孫誘拐事件の教唆を行ったとして見られているそうだが、実際には何もしていなかった、とのこと。",
			NULL, "...Even though people accused Oryou Sonozaki of abetting the kidnapping of Minister Inukai's grandson during the dam conflict five years ago, in reality she didn't have anything to do with it.", Line_Normal);
	ClearMessage();

//rただ、反対運動が最も激化し、園崎家の総力を挙げた一大キャンペーンが最高潮を迎えていたこともあり、自然とその矛先が向けられたことを、逆に利用したそうだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ただ、反対運動が最も激化し、園崎家の総力を挙げた一大キャンペーンが最高潮を迎えていたこともあり、自然とその矛先が向けられたことを、逆に利用したそうだ。",

> 			NULL, "But it massively intensified the opposition movement, and the huge campaign which used all the Sonozaki family's resource was at its peak, so she  subtly cashed in on people's natural denunciation of her.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But it massively intensified the opposition movement, and the huge campaign using all the Sonozaki family's resources was at its peak, so she subtly cashed in on people's natural denunciation of her.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500723", 256, TRUE);
	OutputLine(NULL, "「仕方がないのです。工事現場の監督は、現場の人にも評判が悪かったので口論から発展、と想像できました。",

> 			NULL, "\"It can't be helped. It's easy to imagine the fight with the on-site construction manager developed due to a argument based on his bad relations with the people at the construction site.", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, "\"It can't be helped. It's easy to imagine the fight with the on-site construction manager developed due to an argument based on his bad relations with the people at the construction site.", GetGlobalFlag(GLinemodeSp));
```

				NULL, "\"......A natural death.", Line_WaitForInput);
		ModPlayVoiceLS(4, 5, "ps2/05/210500735", 540, TRUE);
		OutputLine(NULL, "と、言っても信じないのでしょうね」",

> 				NULL, " Even though I said so, you won't believe it, will you?\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Even if I told you that, you wouldn't believe it, would you?\"", Line_Normal);
```

	} else {
		ModPlayVoiceLS(4, 5, "ps3/s14/05/vt4bc_rika1019", 256, TRUE);
		OutputLine(NULL, "「………病死。",
				NULL, "\"......A natural death.", Line_WaitForInput);
		ModPlayVoiceLS(4, 5, "ps3/s14/05/vt4bc_rika1020", 256, TRUE);
		OutputLine(NULL, "と、言っても信じてはくれませんでしょう」",

> 				NULL, " Even though I said so, you won't believe it.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Even if I told you that, you wouldn't believe it, would you?\"", Line_Normal);
```

	if (GetLocalFlag(LConsoleArc)==10) {
		ModPlayVoiceLS(4, 5, "ps2/05/210500736", 540, TRUE);
		OutputLine(NULL, "「でも、まだあなたに話せる時期じゃないから詳しくはいえないけど…祟りでないことは確かよ。",

> 				NULL, "\"It's still not yet time to tell you the details though...anyway, it's certainly not the work of a curse.", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"It's still not yet time to tell you the details, but... it's certainly not the work of a curse.", Line_WaitForInput);
```

//r梨花ちゃんはそれを無造作に受け取ると、ぱらぱらとめくり…そしてその表情はだんだん、不機嫌なものへと変わっていった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "梨花ちゃんはそれを無造作に受け取ると、ぱらぱらとめくり…そしてその表情はだんだん、不機嫌なものへと変わっていった。",

> 			NULL, "Rika-chan took it roughly and began randomly flipping through it... and her mood gradually soured as she did.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Rika-chan roughly grabbed it and began randomly flipping through it... and her mood gradually soured as she did.", Line_Normal);
```
